funnel/models/rsvp.py

```diff
@@ -78,8 +78,8 @@ def response(self):
         message=__("Your response has been saved"),
         type='primary',
     )
-    def rsvp_yes(self):
-        pass
+    def rsvp_yes(self, actor: User):
+        self.project.commentset.add_subscriber(actor=actor, user=actor)
```

Comment:
The actor parameter is not needed, as the subscriber is `rsvp.user`. We also need to make subscription to comments an explicit choice in the UI. This method needs to take a parameter `subscribe_comments=False` and subscribe if True.